src/database/addPost.js

```diff
@@ -1,8 +1,14 @@
 import { db } from "./index";
+import firebase from "firebase";
 
-export const addPost = (body) => {
+export const addPost = (body, opt1, opt2) => {
   db.collection("posts").add({
     body: body,
-    created_at: Date.now(),
+    created_at: firebase.database.ServerValue.TIMESTAMP,
+    option_a_name: opt1,
+    option_b_name: opt2,
+    option_a: 0,
+    option_b: 0,
+    id: "",
```

Comment:
Same naming convention comment as above. Also opt1 and opt2 could have better names

Reply:
This creates them in the DB so Shouldnt I leave it like this?

Reply:
Yeah youre right I missed that, my  bad